web/src/App.js

```diff
@@ -201,13 +198,59 @@ OpenChangesView.propTypes = {
   })
 }
 
+class ChangeView extends React.Component {
+  render () {
+    console.log(this.props.match.params.change)
+    return (
+      <React.Fragment>
+        <TopMenu index={this.props.match.params.index}/>
+        <Container>
+          <Row><Col><p></p></Col></Row>
+          <Row>
+            <Col>
+              <CFiltersForm index={this.props.match.params.index}/>
```

Comment:
Why to keep the filterbox ?

Reply:
I tried to remove it without success so far...

Reply:
ok I got clearer ideas this morning and managed to do it !